test/test_synthetic_images.jl

```diff
@@ -82,7 +82,7 @@ edge_case_imgs = [
             fpath = joinpath(PNG_TEST_PATH, "test_img_$(case).png")
             @testset "write" begin
                 open(io->PNGFiles.save(io, image), fpath, "w") #test IO method
-                @test PNGFiles.save(fpath, image) == 0
+                @test PNGFiles.save(fpath, image) === nothing
```

Comment:
Might this make it a breaking change?

Reply:
We used to always return zero upon successful safe, but, AFAIK, this wasn't intentional; we just implicitly returned a value from libpng. I don't think anyone has a reason to depend on the return value here, but we might cut a "0.4.0" version instead of "0.3.13", if you think it is appropriate.

Reply:
ok. 0.3.13 seems reasonable to me